rosette/rbl/rosette/CMakeLists.txt

```diff
@@ -2,7 +2,8 @@ file(COPY apropos.rbl async-repl.rbl boot.rbl classes-types.rbl
     config.rbl configuration.rbl c-structures.rbl document.rbl
     dyn-load.rbl exceptions.rbl expander.rbl foreign-funs.rbl
     hello_world.rbl io-system.rbl meth-proc-oprns.rbl multimethod.rbl
-    primitives.rbl que-stk-oprns.rbl sbo-init.rbl space.rbl
+    primitives.rbl que-stk-oprns.rbl sbo-init.rbl namespace-util.rbl 
+    namespace.rbl consume.rbl produce.rbl
```

Comment:
Unrelated change.

Reply:
This is needed as space.rbl has been split into multiple files.

Reply:
OK.